front/src/applications/rollingStockEditor/components/RollingStockEditorForm.tsx

```diff
@@ -82,7 +81,7 @@ const RollingStockEditorForm = ({
   const addNewRollingstock = (payload: RollingStockForm) => () => {
     postRollingstock({
       locked: false,
-      rollingStockForm: payload,
+      rollingStockForm: { ...payload, locked: false },
```

Comment:
It seems redundant to pass ` locked: false` both at the root of the payload and within rollingStockForm.

Is there a specific reason for keeping both? Are they meant to serve different purposes at each level of the payload structure?

If not, could we simplify by removing one of them, maybe the one at the root to avoid confusion?